chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/RoomCustomDataSpek.kt

```diff
@@ -0,0 +1,177 @@
+package com.pusher.chatkit
+
+import com.google.common.truth.Truth.assertThat
+import com.pusher.chatkit.Rooms.GENERAL
+import com.pusher.chatkit.Rooms.SAMPLE_CUSTOM_DATA
+import com.pusher.chatkit.Users.ALICE
+import com.pusher.chatkit.Users.PUSHERINO
+import com.pusher.chatkit.Users.SUPER_USER
+import com.pusher.chatkit.test.InstanceActions.createDefaultRole
+import com.pusher.chatkit.test.InstanceActions.newRoom
+import com.pusher.chatkit.test.InstanceActions.newUsers
+import com.pusher.chatkit.test.InstanceSupervisor.setUpInstanceWith
+import com.pusher.chatkit.test.InstanceSupervisor.tearDownInstance
+import com.pusher.chatkit.util.FutureValue
+import junit.framework.Assert.assertNotNull
+import org.jetbrains.spek.api.Spek
```

Comment:
Where there finally any issues in using this move/partial-rewrite as an opportunity to bring Spek 2 here? Just asking as I remember you were enthusiastic about doing it :)